vizro-core/examples/visual-vocabulary/README.md

```diff
@@ -102,13 +96,12 @@ The dashboard is still in development. Below is an overview of the chart types f
 Contributions are welcome! To contribute a chart, follow the steps below:
 
 1. Check that a `svg` file named after the chart type is contained in the [assets](https://github.com/mckinsey/vizro/tree/main/vizro-core/examples/visual-vocabulary/assets/images/charts) folder. If not, [raise an issue](https://github.com/mckinsey/vizro/issues) in the repository.
-2. Add a `.py` file containing a code example of the chart type in the `pages/examples` folder, for instance, `area.py`. Take a look at existing examples.
-3. Create a new page for the chart type and add it to the relevant category `.py` file such as `correlation.py`,
-   `deviation.py`, `distribution.py`, etc. Ensure you add the page to the list of `pages` at the end of the `.py` file.
-4. Remove the `IncompletePage(..)` entry for that chart type in `chart_groups.py`.
-5. Update this `README.md` with the new chart type, its status, category, and API links.
+1. Add a `.py` file containing a code example of the chart type in the `pages/examples` folder, for instance, `area.py`. Take a look at existing examples.
+1. Create a new page for the chart type and add it to the relevant category `.py` file such as `correlation.py`, `deviation.py`, `distribution.py`, etc. Ensure you add the page to the list of `pages` at the end of the `.py` file.
+1. Remove the `IncompletePage(..)` entry for that chart type in `chart_groups.py`.
+1. Update this `README.md` with the new chart type, its status, category, and API links.
```

Comment:
This doesn't like right?

Reply:
It's OK, check the rendered version https://github.com/mckinsey/vizro/blob/cce5730b8c9f97806d03f4c622f5509aa89b5aa9/vizro-core/examples/visual-vocabulary/README.md (it's a markdown syntax to automate bulleted numbering)

Reply:
Ah perfect!